### PR TITLE
prepare changelog for 2.1.0

### PR DESCRIPTION
**Pull request by displague**
Hoping #78 can make it in time. 

Looking to release this May 20, 2021.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+
+DEPRECATIONS:
+- **Deprecated Resource:** `metal_volume` ([#99](https://github.com/equinix/terraform-provider-metal/pull/99))
+- **Deprecated Resource:** `metal_volume_attachment` ([#99](https://github.com/equinix/terraform-provider-metal/pull/99))
+- **Deprecated Data Source:** `metal_volume` ([#99](https://github.com/equinix/terraform-provider-metal/pull/99))
+- removed unintended deprecation of `metal_device`.`facility` ([#82](https://github.com/equinix/terraform-provider-metal/pull/82))
+
+BUG FIXES:
+- removed deprecation of `metal_device`.`facility` ([#82](https://github.com/equinix/terraform-provider-metal/pull/82))
+- fixed plugin crash handling certain errprs ([#89](https://github.com/equinix/terraform-provider-metal/issues/89))
```
Comment (t0mk) [inline CHANGELOG.md:11]:
```suggestion
- fixed plugin crash handling certain errors ([#89](https://github.com/equinix/terraform-provider-metal/issues/89))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,32 @@
...
+
+FEATURES:
+- **New Resource:** `metal_connection` ([#76](https://github.com/equinix/terraform-provider-metal/pull/76))
+- **New Resource:** `metal_virtual_circuit`([#92](https://github.com/equinix/terraform-provider-metal/pull/92))
+- **New Data Source:** `metal_vlan` ([#67](https://github.com/equinix/terraform-provider-metal/pull/67))
+- **New Data Source:** `metal_reserved_ip_blcok`([#80](https://github.com/equinix/terraform-provider-metal/pull/80))
+- **New Data Source:** `metal_port`([#96](https://github.com/equinix/terraform-provider-metal/pull/96))
+- **New Data Source:** `metal_hardware_reservation`([#100](https://github.com/equinix/terraform-provider-metal/pull/100))
+- **New Guide:** Migrating From the Packet Provider ([#72](https://github.com/equinix/terraform-provider-metal/pull/72))
+- **New Guide:** Upgrades Devices from Facilities to Metros ([#103](https://github.com/equinix/terraform-provider-metal/pull/103))
```
Comment (t0mk) [inline CHANGELOG.md:21]:
```suggestion
- **New Guide:** Upgrading Devices from Facilities to Metros ([#103](https://github.com/equinix/terraform-provider-metal/pull/103))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+
+IMPROVEMENTS:
+
+- `metal_project` `md5_pass` field is now flagged sensitive ([#93](https://github.com/equinix/terraform-provider-metal/issues/93))
+- `metal_spot_market_request` `instance_parameters` are documented ([#104](https://github.com/equinix/terraform-provider-metal/issues/104))
+- added `metro` to `metal_connection` (`facility` now optional) ([#81](https://github.com/equinix/terraform-provider-metal/pull/81))
```
Comment (t0mk) [inline CHANGELOG.md:27]:
```suggestion
- added `metro` to `metal_connection` datasource (`facility` now optional) ([#81](https://github.com/equinix/terraform-provider-metal/pull/81))
```

**Review by t0mk — APPROVED** [review]
Looks good.

File: CHANGELOG.md (hunk 3 above)
Comment (displague) [reply CHANGELOG.md:27]: @t0mk thanks - I've picked these suggestions up in https://github.com/equinix/terraform-provider-metal/pull/108

**Comment by displague** [comment]
reopened after closing https://github.com/equinix/terraform-provider-metal/pull/108. The breaking change is deemed minor and compatible with HCL values (`"true"` -> `true`).

**Pull request merged**